src/Domain/FunctionsOfTime/CMakeLists.txt

```diff
@@ -32,4 +35,7 @@ target_link_libraries(
   Utilities
   PRIVATE
   ErrorHandling
+  IO
+  INTERFACE
+  Options
```

Comment:
Move to `PUBLIC` (a source file includes a header that includes Options, which means Options is part of the interface _and_ needed to compile the library, so it should be public)